docsrc/docextract-intro.rst

```diff
@@ -0,0 +1,80 @@
+OCR With DocumentExtraction
+***************************
+
+Introduction
+============
+
+The Indico Platform includes a sophisticated OCR engine that's capable of extracting raw
+text from a variety of document formats including PDF and TIF. OCR functionality is provided
+by the ``DocumentExtraction`` class. Here's how to OCR one document with DocumentExtraction::
+
+    from indico import IndicoClient
+    from indico.queries.documents import DocumentExtraction
+    from indico.queries import JobStatus, RetrieveStorageObject
+
+    # here, IndicoClient assumes you have your api token in your home directory and host set as an env variable
```

Comment:
You may want to split this comment into two lines for better readability (currently creates a scroll bar)